test/modules/linear_boltzmann_solvers/transport_keigen/c5g7/c5g7.lua

```diff
@@ -74,6 +83,30 @@ elseif (k_method == "pi_scdsa_pwlc") then
     })
     solver.Initialize(k_solver)
     solver.Execute(k_solver)
+    elseif (k_method == "pi_smm") then
+        k_solver = lbs.PowerIterationKEigenSMM.Create
+        ({
+            lbs_solver_handle = phys1,
+            accel_pi_verbose = true,
+            k_tol = 1.0e-8,
+            accel_pi_k_tol = 1.0e-8,
+            accel_pi_max_its = 30,
+            diff_sdm = "pwlc",
+        })
+        solver.Initialize(k_solver)
+        solver.Execute(k_solver)
```

Comment:
is it possible to bring these two lines outside of the if statements?

Reply:
I can do that. I did this because it is the way the file existed before I touched it. I will do this for each c5g7 test script.